dashboard/lib/pd/payment/payment_calculator_base.rb

```diff
@@ -62,13 +62,13 @@ def get_pay_period(workshop)
 
     # Determines the payment type.
     # Override in derived classes. Default is either "PLP Urban" or "PLP Non-urban"
-    # for workshops organized by a PLP, otherwise nil.
+    # for workshops organized by a Regional Partner, otherwise nil.
     # @param workshop [Pd::Workshop]
     # @return [String] payment type
     def get_payment_type(workshop)
-      plp = workshop.professional_learning_partner
-      return nil unless plp
-      plp.urban? ? 'PLP Urban' : 'PLP Non-urban'
+      partner = workshop.regional_partner
+      return nil unless partner
+      partner.urban? ? 'PLP Urban' : 'PLP Non-urban'
```

Comment:
Probably, yes. The workshop payment model is changing this sprint and these are going away. I'll take care of it then, if we still have string constants.